plugins/commands/evaluator/src/main/kotlin/EvaluatorCommand.kt

```diff
@@ -317,13 +317,17 @@ class EvaluatorCommand : OrtCommand(
         val scripts = scriptUris.map { it.toURL().readText() }
         val evaluatorRun = evaluator.run(*scripts.toTypedArray())
 
-        val duration = with(evaluatorRun) { Duration.between(startTime, endTime).toKotlinDuration() }
-        echo("The evaluation of ${scriptUris.size} script(s) took $duration.")
+        if (evaluatorRun.violations.isNotEmpty()) {
+            echo("The following ${evaluatorRun.violations.size} rule violations were found:")
```

Comment:
What do you think about inserting: `else echo "No rule violations have been found.`

Should it be `have been` instead of `were` ?